pkg/vulnerabilityreport/builder_test.go

```diff
@@ -105,6 +106,14 @@ func TestScanJobBuilder(t *testing.T) {
 			Get()
 		g.Expect(err).ToNot(gomega.HaveOccurred())
 		g.Expect(job).ToNot(gomega.BeNil())
+
+		specHash := kube.ComputeHash(corev1.PodSpec{
+			Containers: []corev1.Container{{
+				Name:  "nginx",
+				Image: "nginx:1.16",
+			}},
+		})
```

Comment:
While this makes the tests resilient like you mentioned, it also allows the tests to silently pass, even if there's an actual problem because the hash will never be statically defined and always get computed on the fly.

I think it's better to have the hash statically defined, similar to a static golden file. If the tests break and we have to update them often, we should re-think what we are asserting as unit tests for the most part should offer resilience across code changes.

WDYT?

Reply:
it makes sense!
done https://github.com/aquasecurity/trivy-operator/pull/2746/commits/5bc396b191a5c80dce9d2a1e5c63b97f3a7772da